README.md

```diff
@@ -1,90 +1,100 @@
+zopen list --installed
+zopen install which
+zopen list --installed
+which which
+zopen upgrade
+```
 
-But - there is a _transitive closure_ problem to address with porting a software package, namely understanding what packages are pre-requisites
-for the software package you want to port. As an example, for zsh, we see the following:
+## Important usage notes
+- On first run, ```zopen init``` will copy this forked version of meta into the "package" area of zopen (ie. where packages are expanded and accessed from).  It will also be pinned to this release to prevent any possible updates from the "real" meta package. Removing the .pinned file from the meta-dt directory will allow for the main meta port to be installed however this will cause incompatabilities if run.
 
-zsh requires autoconf to configure the build scripts, GNU make to run Makefiles, ncurses
-zsh is easier to develop (but doesn't require) curl and git natively on z/OS, and GNU techinfo for documentation.
+- Remote respositores utilise the suffix ```port``` - where required, packages should be specified **withOUT** the suffix. eg using ```zopen install which``` rather than ```zopen install whichport```
 
-autoconf requires m4, automake, and GNU make
-m4 requires a c99 compiler with c11 features, so make sure you have the latest C/C++ compilers installed on your system.
+## Root filesystem install
+- Selecting ```'/' ``` as the root filesystem will allow the tools to be available system wide for all users who configure their usage. The install needs to be done by a sysadmin [or someone with sufficient rights using the sudo port for example] as the installer will write files to the /usr tree and configuration information to /etc. There will also be a configuration file written as ```/etc/.zopen-config``` - this can be sourced by other users.
+- Removing zopen and the z/OS Open Tools once installed involves: uninstalling all installed packages; removing any copies of ```.zopen-config```; removing the configured zopen root directory (by default ```/usr/local/zopen``` but is set during installation); and then running a command to find any final orphaned symlinks on the system, such as: ```/bin/find $ZOPEN_ROOTFS -type l -exec test ! -e {} \; -print```  where $ZOPEN_ROOTFS is '/' [replace `-print` with `rm -rf` to actually remove symlinks, the example command should only list what was found - care should be taken when removing any files with sysadmin authority to prevent removing critial files!]
 
-GNU make requires GNU m4, automake, autoconf, Perl, and a C compiler that is gcc compatible.
 
-## Order to Build _from scratch_
+## Problem resolution
+If the meta package does get updated to a non-dt version, then running ```./zopen alt meta -s``` from the meta-dt/bin directory within the z/OS Open Tools root filesystem should allow selection of the dt-forked meta; running any other non-forked zopen commands might break the installation or cause unexpected behaviour! 
```

Comment:
Is this now old since ‘dt’ is not a thing ?

Reply:
Updated